### ISSUE TITLE
Problem with strokes

### ISSUE THREAD

**Issue by b4zz4** [chsh2/nijiGPen#1]
Generally works well, but this attribute (layer.active_frame.strokes) seems to not exist in blender 3.2

~~~
Python: Traceback (most recent call last):
  File "/home/bazza/.config/blender/3.2/scripts/addons/nijigp/operator_polygon.py", line 121, in execute
    for j,stroke in enumerate(layer.active_frame.strokes):
AttributeError: 'NoneType' object has no attribute 'strokes'
~~~

**Comment by chsh2** [comment]
I tried to fix this issue in v0.1.1 release. Does it work for you? 
By the way, are you using Blender in Linux? May I ask which specific build you are using?

**Comment by b4zz4** [comment]
Thanks.
Great, work.
I love addon.
